### PR TITLE
Discontinue early

### PR DESCRIPTION
**Pull request by Jngai**
Text change for issue #1134. 

screen shot 2017-10-09 at 10 19 25 am

### CODE REVIEW

File: app/assets/javascripts/student_profile/services_list.jsx [hunk 1]
```diff
@@ -252,9 +252,20 @@ import _ from 'lodash';
       const isHovering = (this.state.hoveringDiscontinueServiceId === service.id);
       const isPending = (this.props.discontinueServiceRequests[service.id] === 'pending');
 
-      const buttonText = (isPending)
-        ? 'Updating...'
-        : (isConfirming) ? 'Confirm' : 'Discontinue Early';
+      var Text = 'Confirm';
+
+      if (isPending) {
```
Comment (Jngai) [inline app/assets/javascripts/student_profile/services_list.jsx]: Button text was already a nested ternary operator. I think writing out the statements is more readable than an even more nested ternary operator. Also `const buttontext` is read only so I added a `text` variable to be modified then assigned to `const buttontext`.

**Comment by Jngai** [comment]
@alexsoble this is good too. Basically I expanded out `buttontext` and added a new condition.

**Comment by Jngai** [comment]
@alexsoble I want to double check this. Will reopen soon.

**Comment by alexsoble** [comment]
@Jngai OK!

**Pull request closed without merging**